library/src/test/java/org/apache/apex/malhar/lib/fs/GenericFileOutputOperatorTest.java

```diff
@@ -166,6 +166,7 @@ public void runTestApplication() throws Exception
     for (int i = 0; i < MAX && (!outputFile.exists()); ++i) {
       Thread.sleep(1000);
     }
+    lc.shutdown();
```

Comment:
I have raised PR for this change separately (APEXMALHAR-2491). I required this for successful travis builds